2024-11-27 18:34:45 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 18:34:45 - INFO - pika.adapters.utils.connection_workflow: Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2024-11-27 18:34:45 - INFO - pika.adapters.utils.io_services_utils: Socket connected: <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 59704), raddr=('127.0.0.1', 5672)>
2024-11-27 18:34:45 - INFO - pika.adapters.utils.connection_workflow: Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x79b01c897bb0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x79b01c897bb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2024-11-27 18:34:45 - INFO - pika.adapters.utils.connection_workflow: AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x79b01c897bb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 18:34:45 - INFO - pika.adapters.utils.connection_workflow: AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x79b01c897bb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 18:34:45 - INFO - pika.adapters.blocking_connection: Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x79b01c897bb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 18:34:45 - INFO - pika.adapters.blocking_connection: Created channel=1
2024-11-27 18:34:45 - INFO - rabbitmq_client.client: Connected to RabbitMQ
2024-11-27 18:34:45 - INFO - rabbitmq_client.client: Exchange 'bews' declared.
2024-11-27 18:34:54 - INFO - rabbitmq_client.client: Restarting application...
2024-11-27 18:34:54 - INFO - rabbitmq_client.client: Closing RabbitMQ connection...
2024-11-27 18:34:54 - INFO - pika.adapters.blocking_connection: Closing connection (200): Normal shutdown
2024-11-27 18:34:54 - INFO - pika.channel: Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x79b01c897bb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 18:34:54 - INFO - pika.channel: Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x79b01c897bb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 18:34:55 - INFO - pika.connection: Closing connection (200): 'Normal shutdown'
2024-11-27 18:34:55 - INFO - pika.adapters.utils.io_services_utils: Aborting transport connection: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 59704), raddr=('127.0.0.1', 5672)>
2024-11-27 18:34:55 - INFO - pika.adapters.utils.io_services_utils: _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 59704), raddr=('127.0.0.1', 5672)>
2024-11-27 18:34:55 - INFO - pika.adapters.utils.io_services_utils: Deactivating transport: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 59704), raddr=('127.0.0.1', 5672)>
2024-11-27 18:34:55 - INFO - pika.connection: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
2024-11-27 18:34:55 - INFO - pika.connection: Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
2024-11-27 18:34:55 - INFO - pika.adapters.utils.io_services_utils: Closing transport socket and unlinking: state=3; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 59704), raddr=('127.0.0.1', 5672)>
2024-11-27 18:34:55 - INFO - pika.adapters.blocking_connection: User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
2024-11-27 18:34:55 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 18:34:55 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 18:34:55 - INFO - pika.adapters.utils.connection_workflow: Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2024-11-27 18:34:55 - INFO - pika.adapters.utils.io_services_utils: Socket connected: <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38128), raddr=('127.0.0.1', 5672)>
2024-11-27 18:34:55 - INFO - pika.adapters.utils.connection_workflow: Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x71b5f0b33bb0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x71b5f0b33bb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2024-11-27 18:34:55 - INFO - pika.adapters.utils.connection_workflow: AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x71b5f0b33bb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 18:34:55 - INFO - pika.adapters.utils.connection_workflow: AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x71b5f0b33bb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 18:34:55 - INFO - pika.adapters.blocking_connection: Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x71b5f0b33bb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 18:34:55 - INFO - pika.adapters.blocking_connection: Created channel=1
2024-11-27 18:34:55 - INFO - rabbitmq_client.client: Connected to RabbitMQ
2024-11-27 18:34:55 - INFO - rabbitmq_client.client: Exchange 'bews' declared.
2024-11-27 18:34:57 - INFO - rabbitmq_client.client: Sent request: 2 with delay: 0 sec
2024-11-27 18:35:49 - INFO - rabbitmq_client.client: Sent request: 2 with delay: 0 sec
2024-11-27 18:35:49 - INFO - rabbitmq_client.client: Received response: 4 for request ID: 322d9d2d-b25e-4f36-8b54-7ba47805e1d4
2024-11-27 18:35:53 - INFO - rabbitmq_client.client: Sent request: 2 with delay: 2 sec
2024-11-27 18:35:57 - INFO - rabbitmq_client.client: Received response: 4 for request ID: 2190b08f-9aee-46fb-8338-3ba4a4344b95
2024-11-27 18:36:06 - INFO - rabbitmq_client.client: Sent request: 5 with delay: 2 sec
2024-11-27 18:36:10 - INFO - rabbitmq_client.client: Received response: 10 for request ID: 8cfb45b6-dce8-49c9-9df8-d18645c42552
2024-11-27 18:36:19 - INFO - rabbitmq_client.client: Closing RabbitMQ connection...
2024-11-27 18:36:19 - INFO - pika.adapters.blocking_connection: Closing connection (200): Normal shutdown
2024-11-27 18:36:19 - INFO - pika.channel: Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x71b5f0b33bb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 18:36:19 - INFO - pika.channel: Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x71b5f0b33bb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 18:36:19 - INFO - pika.connection: Closing connection (200): 'Normal shutdown'
2024-11-27 18:36:19 - INFO - pika.adapters.utils.io_services_utils: Aborting transport connection: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38128), raddr=('127.0.0.1', 5672)>
2024-11-27 18:36:19 - INFO - pika.adapters.utils.io_services_utils: _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38128), raddr=('127.0.0.1', 5672)>
2024-11-27 18:36:19 - INFO - pika.adapters.utils.io_services_utils: Deactivating transport: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38128), raddr=('127.0.0.1', 5672)>
2024-11-27 18:36:19 - INFO - pika.connection: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
2024-11-27 18:36:19 - INFO - pika.connection: Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
2024-11-27 18:36:19 - INFO - pika.adapters.utils.io_services_utils: Closing transport socket and unlinking: state=3; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38128), raddr=('127.0.0.1', 5672)>
2024-11-27 18:36:19 - INFO - pika.adapters.blocking_connection: User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
2024-11-27 18:36:19 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 18:36:19 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:00:18 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:00:58 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:01:54 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:06:15 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:10:12 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:10:12 - INFO - pika.adapters.utils.connection_workflow: Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2024-11-27 19:10:12 - INFO - pika.adapters.utils.io_services_utils: Socket connected: <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 41692), raddr=('127.0.0.1', 5672)>
2024-11-27 19:10:12 - INFO - pika.adapters.utils.connection_workflow: Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7cce3d48be20>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7cce3d48be20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2024-11-27 19:10:12 - INFO - pika.adapters.utils.connection_workflow: AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7cce3d48be20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:10:12 - INFO - pika.adapters.utils.connection_workflow: AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7cce3d48be20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:10:12 - INFO - pika.adapters.blocking_connection: Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7cce3d48be20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:10:12 - INFO - pika.adapters.blocking_connection: Created channel=1
2024-11-27 19:10:12 - INFO - rabbitmq_client.client: Connected to RabbitMQ
2024-11-27 19:10:12 - INFO - rabbitmq_client.client: Exchange 'bews' declared.
2024-11-27 19:10:18 - INFO - rabbitmq_client.client: Sent request: 23 with delay: 2 sec
2024-11-27 19:10:22 - INFO - rabbitmq_client.client: Received response: 46 for request ID: 5fc7cd60-7bbf-4026-9f4d-5a72f89e9e70
2024-11-27 19:10:32 - INFO - rabbitmq_client.client: Closing RabbitMQ connection...
2024-11-27 19:10:32 - INFO - pika.adapters.blocking_connection: Closing connection (200): Normal shutdown
2024-11-27 19:10:32 - ERROR - pika.adapters.utils.io_services_utils: _AsyncBaseTransport._produce() failed, aborting connection: error=IndexError('pop from an empty deque'); sock=<socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 41692), raddr=('127.0.0.1', 5672)>; Caller's stack:
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
2024-11-27 19:10:32 - ERROR - pika.channel: Unexpected frame: <METHOD(['channel_number=1', 'frame_type=1', "method=<Basic.CancelOk(['consumer_tag=ctag1.6ee019dc6c344ea998088efec10290a1'])>"])>
2024-11-27 19:10:32 - INFO - pika.adapters.utils.io_services_utils: _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=IndexError('pop from an empty deque'); <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 41692), raddr=('127.0.0.1', 5672)>
2024-11-27 19:10:32 - INFO - pika.adapters.utils.io_services_utils: Deactivating transport: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 41692), raddr=('127.0.0.1', 5672)>
2024-11-27 19:10:32 - INFO - pika.channel: Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7cce3d48be20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 19:10:32 - ERROR - pika.adapters.base_connection: connection_lost: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:10:32 - INFO - pika.connection: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",); pending-error=None
2024-11-27 19:10:32 - ERROR - rabbitmq_client.client: Error while stopping client: 'NoneType' object has no attribute 'write'
2024-11-27 19:10:32 - INFO - pika.connection: Stack terminated due to StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:10:32 - INFO - pika.adapters.utils.io_services_utils: Closing transport socket and unlinking: state=2; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 41692), raddr=('127.0.0.1', 5672)>
2024-11-27 19:10:32 - ERROR - pika.adapters.blocking_connection: Unexpected connection close detected: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:10:32 - ERROR - rabbitmq_client.client: Error in client run loop: Stream connection lost: IndexError('pop from an empty deque')
2024-11-27 19:10:32 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:10:32 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:10:35 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:10:35 - INFO - pika.adapters.utils.connection_workflow: Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2024-11-27 19:10:35 - INFO - pika.adapters.utils.io_services_utils: Socket connected: <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58190), raddr=('127.0.0.1', 5672)>
2024-11-27 19:10:35 - INFO - pika.adapters.utils.connection_workflow: Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7e16289338b0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7e16289338b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2024-11-27 19:10:35 - INFO - pika.adapters.utils.connection_workflow: AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7e16289338b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:10:35 - INFO - pika.adapters.utils.connection_workflow: AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7e16289338b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:10:35 - INFO - pika.adapters.blocking_connection: Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7e16289338b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:10:35 - INFO - pika.adapters.blocking_connection: Created channel=1
2024-11-27 19:10:35 - INFO - rabbitmq_client.client: Connected to RabbitMQ
2024-11-27 19:10:35 - INFO - rabbitmq_client.client: Exchange 'bews' declared.
2024-11-27 19:10:39 - INFO - rabbitmq_client.client: Sent request: 2 with delay: 2 sec
2024-11-27 19:10:50 - INFO - rabbitmq_client.client: Closing RabbitMQ connection...
2024-11-27 19:10:50 - INFO - pika.adapters.blocking_connection: Closing connection (200): Normal shutdown
2024-11-27 19:10:50 - ERROR - pika.adapters.utils.io_services_utils: _AsyncBaseTransport._produce() failed, aborting connection: error=IndexError('pop from an empty deque'); sock=<socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58190), raddr=('127.0.0.1', 5672)>; Caller's stack:
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
2024-11-27 19:10:50 - ERROR - pika.channel: Unexpected frame: <METHOD(['channel_number=1', 'frame_type=1', "method=<Basic.CancelOk(['consumer_tag=ctag1.3eb8bcb4189c4fa9ac80f5a83a0a6cdc'])>"])>
2024-11-27 19:10:50 - INFO - pika.adapters.utils.io_services_utils: _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=IndexError('pop from an empty deque'); <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58190), raddr=('127.0.0.1', 5672)>
2024-11-27 19:10:50 - INFO - pika.adapters.utils.io_services_utils: Deactivating transport: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58190), raddr=('127.0.0.1', 5672)>
2024-11-27 19:10:50 - ERROR - pika.adapters.base_connection: connection_lost: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:10:50 - INFO - pika.channel: Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 19:10:50 - INFO - pika.connection: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",); pending-error=None
2024-11-27 19:10:50 - ERROR - rabbitmq_client.client: Error while stopping client: 'NoneType' object has no attribute 'write'
2024-11-27 19:10:50 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:10:50 - INFO - pika.connection: Stack terminated due to StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:10:50 - INFO - pika.adapters.utils.io_services_utils: Closing transport socket and unlinking: state=2; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58190), raddr=('127.0.0.1', 5672)>
2024-11-27 19:10:50 - ERROR - pika.adapters.blocking_connection: Unexpected connection close detected: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:10:50 - ERROR - rabbitmq_client.client: Error in client run loop: Stream connection lost: IndexError('pop from an empty deque')
2024-11-27 19:10:50 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:11:35 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:12:03 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:12:03 - INFO - pika.adapters.utils.connection_workflow: Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2024-11-27 19:12:03 - INFO - pika.adapters.utils.io_services_utils: Socket connected: <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 43492), raddr=('127.0.0.1', 5672)>
2024-11-27 19:12:03 - INFO - pika.adapters.utils.connection_workflow: Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x72516a72beb0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x72516a72beb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2024-11-27 19:12:03 - INFO - pika.adapters.utils.connection_workflow: AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x72516a72beb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:12:03 - INFO - pika.adapters.utils.connection_workflow: AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x72516a72beb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:12:03 - INFO - pika.adapters.blocking_connection: Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x72516a72beb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:12:03 - INFO - pika.adapters.blocking_connection: Created channel=1
2024-11-27 19:12:03 - INFO - rabbitmq_client.client: Connected to RabbitMQ
2024-11-27 19:12:03 - INFO - rabbitmq_client.client: Exchange 'bews' declared.
2024-11-27 19:12:10 - INFO - rabbitmq_client.client: Sent request: 23323213 with delay: 3 sec
2024-11-27 19:13:25 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:13:40 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:13:53 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:13:53 - INFO - pika.adapters.utils.connection_workflow: Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2024-11-27 19:13:53 - INFO - pika.adapters.utils.io_services_utils: Socket connected: <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 44186), raddr=('127.0.0.1', 5672)>
2024-11-27 19:13:53 - INFO - pika.adapters.utils.connection_workflow: Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7d48c0927e20>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7d48c0927e20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2024-11-27 19:13:53 - INFO - pika.adapters.utils.connection_workflow: AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7d48c0927e20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:13:53 - INFO - pika.adapters.utils.connection_workflow: AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7d48c0927e20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:13:53 - INFO - pika.adapters.blocking_connection: Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7d48c0927e20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:13:53 - INFO - pika.adapters.blocking_connection: Created channel=1
2024-11-27 19:13:53 - INFO - rabbitmq_client.client: Connected to RabbitMQ
2024-11-27 19:13:53 - INFO - rabbitmq_client.client: Exchange 'bews' declared.
2024-11-27 19:14:03 - INFO - rabbitmq_client.client: Restarting application...
2024-11-27 19:14:03 - INFO - rabbitmq_client.client: Closing RabbitMQ connection...
2024-11-27 19:14:03 - INFO - pika.adapters.blocking_connection: Closing connection (200): Normal shutdown
2024-11-27 19:14:03 - ERROR - pika.adapters.utils.io_services_utils: _AsyncBaseTransport._produce() failed, aborting connection: error=IndexError('pop from an empty deque'); sock=<socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 44186), raddr=('127.0.0.1', 5672)>; Caller's stack:
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
2024-11-27 19:14:03 - ERROR - pika.channel: Unexpected frame: <METHOD(['channel_number=1', 'frame_type=1', "method=<Basic.CancelOk(['consumer_tag=ctag1.e6c65c03fae14305ba2ec2e907bca471'])>"])>
2024-11-27 19:14:03 - INFO - pika.adapters.utils.io_services_utils: _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=IndexError('pop from an empty deque'); <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 44186), raddr=('127.0.0.1', 5672)>
2024-11-27 19:14:03 - INFO - pika.channel: Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7d48c0927e20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 19:14:03 - INFO - pika.adapters.utils.io_services_utils: Deactivating transport: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 44186), raddr=('127.0.0.1', 5672)>
2024-11-27 19:14:03 - ERROR - pika.adapters.base_connection: connection_lost: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:14:03 - INFO - pika.connection: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",); pending-error=None
2024-11-27 19:14:03 - INFO - pika.connection: Stack terminated due to StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:14:03 - INFO - pika.adapters.utils.io_services_utils: Closing transport socket and unlinking: state=2; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 44186), raddr=('127.0.0.1', 5672)>
2024-11-27 19:14:03 - ERROR - pika.adapters.blocking_connection: Unexpected connection close detected: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:14:03 - ERROR - rabbitmq_client.client: Error in client run loop: Stream connection lost: IndexError('pop from an empty deque')
2024-11-27 19:14:04 - ERROR - rabbitmq_client.client: Error while stopping client: 'NoneType' object has no attribute 'clear'
2024-11-27 19:14:04 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:14:04 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:14:04 - INFO - pika.adapters.utils.connection_workflow: Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2024-11-27 19:14:04 - INFO - pika.adapters.utils.io_services_utils: Socket connected: <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 54268), raddr=('127.0.0.1', 5672)>
2024-11-27 19:14:04 - INFO - pika.adapters.utils.connection_workflow: Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76980a127eb0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76980a127eb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2024-11-27 19:14:04 - INFO - pika.adapters.utils.connection_workflow: AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76980a127eb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:14:04 - INFO - pika.adapters.utils.connection_workflow: AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76980a127eb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:14:04 - INFO - pika.adapters.blocking_connection: Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76980a127eb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:14:04 - INFO - pika.adapters.blocking_connection: Created channel=1
2024-11-27 19:14:04 - INFO - rabbitmq_client.client: Connected to RabbitMQ
2024-11-27 19:14:04 - INFO - rabbitmq_client.client: Exchange 'bews' declared.
2024-11-27 19:14:09 - INFO - rabbitmq_client.client: Sent request: 2 with delay: 2 sec
2024-11-27 19:14:21 - INFO - rabbitmq_client.client: Closing RabbitMQ connection...
2024-11-27 19:14:21 - INFO - pika.adapters.blocking_connection: Closing connection (200): Normal shutdown
2024-11-27 19:14:21 - INFO - pika.channel: Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76980a127eb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 19:14:21 - INFO - pika.channel: Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76980a127eb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 19:14:21 - INFO - pika.connection: Closing connection (200): 'Normal shutdown'
2024-11-27 19:14:21 - INFO - pika.adapters.utils.io_services_utils: Aborting transport connection: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 54268), raddr=('127.0.0.1', 5672)>
2024-11-27 19:14:21 - INFO - pika.adapters.utils.io_services_utils: _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 54268), raddr=('127.0.0.1', 5672)>
2024-11-27 19:14:21 - INFO - pika.adapters.utils.io_services_utils: Deactivating transport: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 54268), raddr=('127.0.0.1', 5672)>
2024-11-27 19:14:21 - INFO - pika.connection: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
2024-11-27 19:14:21 - INFO - pika.connection: Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
2024-11-27 19:14:21 - INFO - pika.adapters.utils.io_services_utils: Closing transport socket and unlinking: state=3; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 54268), raddr=('127.0.0.1', 5672)>
2024-11-27 19:14:21 - INFO - pika.adapters.blocking_connection: User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
2024-11-27 19:14:21 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:14:21 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:19:02 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:22:25 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:22:25 - INFO - pika.adapters.utils.connection_workflow: Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2024-11-27 19:22:25 - INFO - pika.adapters.utils.io_services_utils: Socket connected: <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58338), raddr=('127.0.0.1', 5672)>
2024-11-27 19:22:25 - INFO - pika.adapters.utils.connection_workflow: Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7a232529fe20>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7a232529fe20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2024-11-27 19:22:25 - INFO - pika.adapters.utils.connection_workflow: AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7a232529fe20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:22:25 - INFO - pika.adapters.utils.connection_workflow: AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7a232529fe20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:22:25 - INFO - pika.adapters.blocking_connection: Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7a232529fe20> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:22:25 - INFO - pika.adapters.blocking_connection: Created channel=1
2024-11-27 19:22:25 - INFO - rabbitmq_client.client: Connected to RabbitMQ
2024-11-27 19:22:25 - INFO - rabbitmq_client.client: Exchange 'bews' declared.
2024-11-27 19:22:30 - INFO - rabbitmq_client.client: Restarting application...
2024-11-27 19:22:30 - INFO - rabbitmq_client.client: Closing RabbitMQ connection...
2024-11-27 19:22:30 - INFO - pika.adapters.blocking_connection: Closing connection (200): Normal shutdown
2024-11-27 19:22:30 - ERROR - pika.channel: Unexpected frame: <METHOD(['channel_number=1', 'frame_type=1', "method=<Basic.CancelOk(['consumer_tag=ctag1.b752cf2a06044093834a759ff62a47b2'])>"])>
2024-11-27 19:22:30 - ERROR - pika.adapters.utils.io_services_utils: _AsyncBaseTransport._produce() failed, aborting connection: error=IndexError('pop from an empty deque'); sock=<socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58338), raddr=('127.0.0.1', 5672)>; Caller's stack:
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
2024-11-27 19:22:30 - INFO - pika.adapters.utils.io_services_utils: _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=IndexError('pop from an empty deque'); <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58338), raddr=('127.0.0.1', 5672)>
2024-11-27 19:22:30 - INFO - pika.adapters.utils.io_services_utils: Deactivating transport: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58338), raddr=('127.0.0.1', 5672)>
2024-11-27 19:22:30 - ERROR - pika.adapters.base_connection: connection_lost: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:22:30 - INFO - pika.connection: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",); pending-error=None
2024-11-27 19:22:30 - INFO - pika.channel: Closing channel (200): 'Normal shutdown' on <Channel number=1 CLOSED conn=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 19:22:30 - INFO - pika.connection: Stack terminated due to StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:22:30 - ERROR - pika.connection: Attempted to send frame when closed
2024-11-27 19:22:30 - INFO - pika.adapters.utils.io_services_utils: Closing transport socket and unlinking: state=2; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58338), raddr=('127.0.0.1', 5672)>
2024-11-27 19:22:30 - ERROR - rabbitmq_client.client: Error while stopping client: Attempted to send a frame on closed connection.
2024-11-27 19:22:30 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:22:30 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:22:30 - INFO - pika.adapters.utils.connection_workflow: Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2024-11-27 19:22:30 - INFO - pika.adapters.utils.io_services_utils: Socket connected: <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58528), raddr=('127.0.0.1', 5672)>
2024-11-27 19:22:30 - INFO - pika.adapters.utils.connection_workflow: Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7a70115278b0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7a70115278b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2024-11-27 19:22:30 - INFO - pika.adapters.utils.connection_workflow: AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7a70115278b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:22:30 - INFO - pika.adapters.utils.connection_workflow: AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7a70115278b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:22:30 - INFO - pika.adapters.blocking_connection: Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7a70115278b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:22:30 - INFO - pika.adapters.blocking_connection: Created channel=1
2024-11-27 19:22:30 - INFO - rabbitmq_client.client: Connected to RabbitMQ
2024-11-27 19:22:30 - INFO - rabbitmq_client.client: Exchange 'bews' declared.
2024-11-27 19:22:44 - INFO - rabbitmq_client.client: Closing RabbitMQ connection...
2024-11-27 19:22:44 - INFO - pika.adapters.blocking_connection: Closing connection (200): Normal shutdown
2024-11-27 19:22:44 - ERROR - pika.channel: Unexpected frame: <METHOD(['channel_number=1', 'frame_type=1', "method=<Basic.CancelOk(['consumer_tag=ctag1.2186732f4395472692ec7a14cc028bf1'])>"])>
2024-11-27 19:22:44 - INFO - pika.channel: Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7a70115278b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 19:22:44 - ERROR - pika.adapters.utils.io_services_utils: _AsyncBaseTransport._produce() failed, aborting connection: error=IndexError('pop from an empty deque'); sock=<socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58528), raddr=('127.0.0.1', 5672)>; Caller's stack:
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
2024-11-27 19:22:44 - INFO - pika.channel: Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7a70115278b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 19:22:44 - INFO - pika.adapters.utils.io_services_utils: _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=IndexError('pop from an empty deque'); <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58528), raddr=('127.0.0.1', 5672)>
2024-11-27 19:22:44 - INFO - pika.connection: Closing connection (200): 'Normal shutdown'
2024-11-27 19:22:44 - INFO - pika.adapters.utils.io_services_utils: Deactivating transport: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58528), raddr=('127.0.0.1', 5672)>
2024-11-27 19:22:44 - ERROR - pika.adapters.base_connection: connection_lost: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:22:44 - INFO - pika.connection: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",); pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
2024-11-27 19:22:44 - INFO - pika.connection: Stack terminated due to StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:22:44 - INFO - pika.adapters.utils.io_services_utils: Closing transport socket and unlinking: state=2; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 58528), raddr=('127.0.0.1', 5672)>
2024-11-27 19:22:44 - ERROR - pika.adapters.blocking_connection: Unexpected connection close detected: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:22:44 - ERROR - pika.adapters.blocking_connection: Unexpected connection close detected: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:22:44 - ERROR - rabbitmq_client.client: Stream connection lost while closing: Stream connection lost: IndexError('pop from an empty deque')
2024-11-27 19:22:44 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:22:44 - ERROR - rabbitmq_client.client: Error in client run loop: Stream connection lost: IndexError('pop from an empty deque')
2024-11-27 19:22:44 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:23:33 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:23:33 - INFO - pika.adapters.utils.connection_workflow: Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2024-11-27 19:23:33 - INFO - pika.adapters.utils.io_services_utils: Socket connected: <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 54690), raddr=('127.0.0.1', 5672)>
2024-11-27 19:23:33 - INFO - pika.adapters.utils.connection_workflow: Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76d24849feb0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76d24849feb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2024-11-27 19:23:33 - INFO - pika.adapters.utils.connection_workflow: AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76d24849feb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:23:33 - INFO - pika.adapters.utils.connection_workflow: AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76d24849feb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:23:33 - INFO - pika.adapters.blocking_connection: Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76d24849feb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:23:33 - INFO - pika.adapters.blocking_connection: Created channel=1
2024-11-27 19:23:33 - INFO - rabbitmq_client.client: Connected to RabbitMQ
2024-11-27 19:23:33 - INFO - rabbitmq_client.client: Exchange 'bews' declared.
2024-11-27 19:23:38 - INFO - rabbitmq_client.client: Sent request: 2 with delay: 2 sec
2024-11-27 19:23:47 - INFO - rabbitmq_client.client: Closing RabbitMQ connection...
2024-11-27 19:23:47 - INFO - pika.adapters.blocking_connection: Closing connection (200): Normal shutdown
2024-11-27 19:23:47 - ERROR - pika.adapters.utils.io_services_utils: _AsyncBaseTransport._produce() failed, aborting connection: error=IndexError('pop from an empty deque'); sock=<socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 54690), raddr=('127.0.0.1', 5672)>; Caller's stack:
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
2024-11-27 19:23:47 - ERROR - pika.channel: Unexpected frame: <METHOD(['channel_number=1', 'frame_type=1', "method=<Basic.CancelOk(['consumer_tag=ctag1.69bf349cfefe4ec49374102bc891e0d3'])>"])>
2024-11-27 19:23:47 - INFO - pika.adapters.utils.io_services_utils: _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=IndexError('pop from an empty deque'); <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 54690), raddr=('127.0.0.1', 5672)>
2024-11-27 19:23:47 - INFO - pika.adapters.utils.io_services_utils: Deactivating transport: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 54690), raddr=('127.0.0.1', 5672)>
2024-11-27 19:23:47 - INFO - pika.channel: Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76d24849feb0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 19:23:47 - ERROR - pika.adapters.base_connection: connection_lost: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:23:47 - ERROR - rabbitmq_client.client: Error while stopping client: 'NoneType' object has no attribute 'write'
2024-11-27 19:23:47 - INFO - pika.connection: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",); pending-error=None
2024-11-27 19:23:47 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:23:47 - INFO - pika.connection: Stack terminated due to StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:23:47 - INFO - pika.adapters.utils.io_services_utils: Closing transport socket and unlinking: state=2; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 54690), raddr=('127.0.0.1', 5672)>
2024-11-27 19:23:47 - ERROR - pika.adapters.blocking_connection: Unexpected connection close detected: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:23:47 - ERROR - rabbitmq_client.client: Error in client run loop: Stream connection lost: IndexError('pop from an empty deque')
2024-11-27 19:23:47 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:24:12 - INFO - rabbitmq_client.client: Logging initialized for RMQClient.
2024-11-27 19:24:13 - INFO - pika.adapters.utils.connection_workflow: Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2024-11-27 19:24:13 - INFO - pika.adapters.utils.io_services_utils: Socket connected: <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 36462), raddr=('127.0.0.1', 5672)>
2024-11-27 19:24:13 - INFO - pika.adapters.utils.connection_workflow: Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7050565402b0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7050565402b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2024-11-27 19:24:13 - INFO - pika.adapters.utils.connection_workflow: AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7050565402b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:24:13 - INFO - pika.adapters.utils.connection_workflow: AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7050565402b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:24:13 - INFO - pika.adapters.blocking_connection: Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7050565402b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2024-11-27 19:24:13 - INFO - pika.adapters.blocking_connection: Created channel=1
2024-11-27 19:24:13 - INFO - rabbitmq_client.client: Connected to RabbitMQ
2024-11-27 19:24:13 - INFO - rabbitmq_client.client: Exchange 'bews' declared.
2024-11-27 19:24:16 - INFO - rabbitmq_client.client: Sent request: 2 with delay: 0 sec
2024-11-27 19:24:40 - INFO - rabbitmq_client.client: Sent request: 2 with delay: 0 sec
2024-11-27 19:24:40 - INFO - rabbitmq_client.client: Received response: 4 for request ID: e2e7cb09-df0b-40d8-98ff-27e426a16804
2024-11-27 19:24:47 - INFO - rabbitmq_client.client: Closing RabbitMQ connection...
2024-11-27 19:24:47 - INFO - pika.adapters.blocking_connection: Closing connection (200): Normal shutdown
2024-11-27 19:24:47 - ERROR - pika.channel: Unexpected frame: <METHOD(['channel_number=1', 'frame_type=1', "method=<Basic.CancelOk(['consumer_tag=ctag1.288dac59763d4071b75a50fc6128361a'])>"])>
2024-11-27 19:24:47 - INFO - pika.channel: Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7050565402b0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2024-11-27 19:24:47 - ERROR - pika.adapters.utils.io_services_utils: _AsyncBaseTransport._produce() failed, aborting connection: error=IndexError('pop from an empty deque'); sock=<socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 36462), raddr=('127.0.0.1', 5672)>; Caller's stack:
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
Traceback (most recent call last):
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 1103, in _on_socket_writable
    self._produce()
  File "/home/user/Desktop/GUI 2222/venv/lib/python3.10/site-packages/pika/adapters/utils/io_services_utils.py", line 822, in _produce
    chunk = self._tx_buffers.popleft()
IndexError: pop from an empty deque
2024-11-27 19:24:47 - INFO - pika.adapters.utils.io_services_utils: _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=IndexError('pop from an empty deque'); <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 36462), raddr=('127.0.0.1', 5672)>
2024-11-27 19:24:47 - INFO - pika.adapters.utils.io_services_utils: Deactivating transport: state=1; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 36462), raddr=('127.0.0.1', 5672)>
2024-11-27 19:24:47 - ERROR - pika.adapters.base_connection: connection_lost: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:24:47 - INFO - pika.connection: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",); pending-error=None
2024-11-27 19:24:47 - INFO - pika.connection: Stack terminated due to StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:24:47 - INFO - pika.adapters.utils.io_services_utils: Closing transport socket and unlinking: state=2; <socket.socket fd=15, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 36462), raddr=('127.0.0.1', 5672)>
2024-11-27 19:24:47 - ERROR - pika.adapters.blocking_connection: Unexpected connection close detected: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:24:47 - ERROR - pika.adapters.blocking_connection: Unexpected connection close detected: StreamLostError: ("Stream connection lost: IndexError('pop from an empty deque')",)
2024-11-27 19:24:47 - ERROR - rabbitmq_client.client: Error while stopping client: _CallbackResult was not set
2024-11-27 19:24:47 - INFO - rabbitmq_client.client: Client stopped successfully.
2024-11-27 19:24:47 - ERROR - rabbitmq_client.client: Error in client run loop: Stream connection lost: IndexError('pop from an empty deque')
2024-11-27 19:24:47 - INFO - rabbitmq_client.client: Client stopped successfully.
